analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -191,6 +191,8 @@ abstract class PackageManager(
                 } catch (e: Exception) {
                     e.showStackTrace()
 
+                    log.error { "Resolving dependencies for '${definitionFile.name}' failed with: ${e.message}" }
```

Comment:
Not sure if this is out of scope: for the pattern you consistently establish here, would it be possible to reduce the amount of calls from two to one? One option could be to overload log.error as

log.error( message: String, e :  Throwable) {
     e.showstacktrace
     log.error( message )
}

Acknowledge that it''s also not 100% clean to have this showstacktrace call inside a log.* function,
on the positive side it gives more compact client code.

Reply:
I plan to do something like this in a separate PR, likely as extension function for `Throwable`.